### DISCUSSION TITLE
6.3.3 in Chrome stuck on "Loading"

### DISCUSSION THREAD

**Discussion by wmertens** [Authenticator-Extension/Authenticator#824] [Q&A]
### Describe the issue

When I try to unlock the extension, it shows "Loading" but nothing else. The background page doesn't show an error.

Version is 6.3.3, the latest in the Chrome web store.

How can I extract a backup in this state?

### Browser

Chrome

### Browser Version

96

### Extension Version

6.3.3

**Comment by mymindstorm** [comment]
Please follow [these steps](https://authenticator.cc/docs/en/debug-info) and reproduce the issue.

**Comment by wmertens** [comment]
There are no error messages, only this:
```
Add Dragula plugin: Object
DevTools failed to load source map: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/dist/popup.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://bhghoamapcdpbohphigoooaddinpkbai/css/popup.css.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
```

Maybe I can breakpoint somewhere interesting in the code?

**Reply by Sneezry** [reply]
You can turn on **Collect errors** and check if there is any runtime error when the extension is stuck on "Loading"

![image](https://user-images.githubusercontent.com/1621293/149323156-16cab24c-9f5f-4154-ad9e-26277c27bafb.png)

![image](https://user-images.githubusercontent.com/1621293/149323182-f57d0cb1-b358-4d42-8200-dbf4a3f8f865.png)

![image](https://user-images.githubusercontent.com/1621293/149323205-15a69b83-1d3b-42ee-9b63-c700ecd59468.png)

**Reply by wmertens** [reply]
Alas, no change

**Reply by Sneezry** [reply]
Have you enabled browser sync?

**Reply by Sneezry** [reply]
If you have enabled browser sync, try to run `chrome.storage.sync.get(d=>console.log(JSON.stringify(d)))` in console to get your data by following [this doc](https://authenticator.cc/docs/en/debug-info). If you have disabled browser sync, try to run `chrome.storage.local.get(d=>console.log(JSON.stringify(d)))` to get your data.

![image](https://user-images.githubusercontent.com/1621293/149349212-db908c24-6b51-4ea8-add6-6d08363251f0.png)

![image](https://user-images.githubusercontent.com/1621293/149349167-1e7c642c-6a78-466d-8708-3c79d2d323f7.png)

If your data is encrypted, you are not able to import the data to other 2FA clients directly. To decrypt the data, you can open a newly installed Authenticator extension (you can create a new profile in the browser, and install the extension with that profile browser window), open import/export page, switch to Import Text Backup, and paste all the text you get in the console. Then check Encrypted, and type the password, click Import Text Backup. After that, you can export the backup without a password and transfer the data to another 2FA client or your previous Authenticator extension.

![image](https://user-images.githubusercontent.com/1621293/149350181-33ec8750-cf34-4b8d-8dc8-8fc265358c7a.png)

**Reply by mymindstorm** [reply]
> If your data is encrypted, you are not able to import the data to other 2FA clients directly. To decrypt the data, you can open a newly installed Authenticator extension (you can create a new profile in the browser, and install the extension with that profile browser window)

You can also use `chrome://extensions/?id=bhghoamapcdpbohphigoooaddinpkbai` -> "Extension Options" to reset the extension.

**Reply by wmertens** [reply]
I finally got around to trying this, thanks. It looks like I don't have any data synced (both local and sync return `undefined`), and maybe that's why it behaves that way. So somehow sync deleted the storage and since then it's been hanging on unlock?
